docker-compose-fuelrod.yml

```
@@ -5,12 +5,15 @@ include:
   - compose/docker-compose.base.yml
   - compose/services/docker-compose.database.yml
   - compose/services/docker-compose.redis.yml
+  #- compose/services/docker-compose.redis-admin.yml
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)